config/config.go

```diff
@@ -230,20 +230,25 @@ type DeploymentConfig struct {
 	AccessibleNamespaces []string `yaml:"accessible_namespaces"`
 }
 
+// IstioComponentNamespaces holds the component-specific Istio namespaces. Any mising component
+// defaults to the namespace configured for IstioNamespace (which itself defaults to 'istio-system').
+type IstioComponentNamespaces map[string]string
+
 // Config defines full YAML configuration.
 type Config struct {
-	Identity         security.Identity `yaml:",omitempty"`
-	Server           Server            `yaml:",omitempty"`
-	InCluster        bool              `yaml:"in_cluster,omitempty"`
-	ExternalServices ExternalServices  `yaml:"external_services,omitempty"`
-	LoginToken       LoginToken        `yaml:"login_token,omitempty"`
-	IstioNamespace   string            `yaml:"istio_namespace,omitempty"`
-	InstallationTag  string            `yaml:"installation_tag,omitempty"`
-	IstioLabels      IstioLabels       `yaml:"istio_labels,omitempty"`
-	KubernetesConfig KubernetesConfig  `yaml:"kubernetes_config,omitempty"`
-	API              ApiConfig         `yaml:"api,omitempty"`
-	Auth             AuthConfig        `yaml:"auth,omitempty"`
-	Deployment       DeploymentConfig  `yaml:"deployment,omitempty"`
+	Identity         security.Identity        `yaml:",omitempty"`
+	Server           Server                   `yaml:",omitempty"`
+	InCluster        bool                     `yaml:"in_cluster,omitempty"`
+	ExternalServices ExternalServices         `yaml:"external_services,omitempty"`
+	LoginToken       LoginToken               `yaml:"login_token,omitempty"`
+	IstioNamespace   string                   `yaml:"istio_namespace,omitempty"` // default component namespace
+	IstioNamespaces  IstioComponentNamespaces `yaml:"istio_namespaces,omitempty"`
```

Comment:
yaml is istio_component_namespaces
```suggestion
	IstioNamespaces  IstioComponentNamespaces `yaml:"istio_component_namespaces,omitempty"`
```

Reply:
Yes, I had already changed it locally, will be updated in next commit.